sycl/include/CL/sycl/handler.hpp

```diff
@@ -809,13 +788,10 @@ class __SYCL_EXPORT handler {
     // call-graph to make this_item calls kernel-specific but this is
     // not considered worthwhile.
 
-    // Get the kernal name to check condition 3.
+    // Get the kernal name to check condition 1.
     std::string KName = typeid(NameT *).name();
     using KI = detail::KernelInfo<KernelName>;
     bool DisableRounding =
-        (getenv("SYCL_DISABLE_PARALLEL_FOR_RANGE_ROUNDING") != nullptr) ||
```

Comment:
This environment variable is documented.
I don't know if there are users of this environment variable.
Anyway, I believe, it should become deprecated first with or without any effect.
Also, if the reason for eliminating of this env var is reducing host overhead, one could employ `SYCLConfig` class from `source/detail/config.hpp`.